lbrynet/core/Wallet.py

```diff
@@ -740,6 +740,9 @@ def _decode_and_cache_claim_result(self, claim, update_caches):
             name = claim['name']
             claim['value'] = claim_dict
             claim['hex'] = decoded.serialized.encode('hex')
+
+            # lbry/issue#958
+            claim['permanent_url'] = claim["name"] + "#" + claim["claim_id"]
```

Comment:
These should use [formatted strings](https://docs.python.org/2/library/stdtypes.html#string-formatting), like `           claim['permanent_url'] = "%s#%s" % (claim['name'], claim['claim_id')`

Reply:
These should really go in lbryum ([here](https://github.com/lbryio/lbryum/blob/f7fd95f54687c46911471f3d805a155209701a53/lbryum/commands.py#L1093) for the function that formats the response for `getvalueforname` (winning resolution) and [here](https://github.com/lbryio/lbryum/blob/f7fd95f54687c46911471f3d805a155209701a53/lbryum/commands.py#L994) for the rest)

Reply:
The channel name can also be included in the permanent url (as requested by @kauffj) in lbryum if the signature is valid.

Reply:
For even better forward compatibility (python 3) use the `.format()` string method:
```
claim['permanent_url'] = "{}#{}".format(claim["name"], claim["claim_id"])
```